erc20-bridge-token/contracts/ProofConsumer.sol

```diff
@@ -21,17 +21,17 @@ contract ProofConsumer {
     mapping(bytes32 => bool) public usedProofs;
 
     constructor (
-        bytes memory _nearTokenFactory,
+        bytes memory _nearTokenLocker,
         INearProver _prover,
         uint64 _minBlockAcceptanceHeight
     )  {
         require(
-            _nearTokenFactory.length > 0,
+            _nearTokenLocker.length > 0,
             "Invalid Near Token Factory address"
```

Comment:
As we are changing the naming, we should also change the error message 
```suggestion
            "Invalid Near Token Locker address"
```

Reply:
Fixed c112a8c38f43c876660c52e06a601a6ee01281f6